apps/src/templates/CensusForm.jsx

```diff
@@ -90,6 +112,7 @@ class CensusForm extends Component {
         <div style={styles.options}>
           {CSOptions.map((label, index) =>
             <Checkbox
+              name={`${label}_b`}
```

Comment:
I'm kind of stuck on how to post data about the checkboxes. I need the state (unchecked/checked) in the `Checkbox` component to be translated into state, something like `pledge : true` or `pledge : false`, in the state of the form.  Any thoughts?

Reply:
It looks like you currently use $('#census-form').serialize() to decide what to post? I'm not too sure exactly how this works, so I'm not sure how to get it to do what you want. One option though might to build your own serializing method that looks at your component state and/or DOM state in order to decide what to post.

Reply:
I suspect the jquery.serialize approach is only going to work with native checkboxes. it's possible you could put a hidden checkbox input inside the Checkbox component, and keep its value up to date with the React component's state.

Reply:
I only made the `Checkbox` component so I could iterate through that array to make a bunch. I could just nix that sub component and make them more like I make the drop down options for role and keep it all the state in the form?